tron/bin/check_tron_datastore_staleness.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python3.6
+import argparse
+import logging
+import os
+import sys
+import time
+
+import pytz
+
+from tron.config import manager
+from tron.config import schema
+from tron.serialize.runstate.dynamodb_state_store import DynamoDBStateStore
+
+# Default values for arguments
+DEFAULT_WORKING_DIR = '/var/lib/tron/'
+DEFAULT_CONF_PATH = 'config/'
+DEFAULT_STALENESS_THRESHOLD = 30
+log = logging.getLogger('tron_stateless_alert')
+
+
+def get_last_run_time(job):
+    '''
+    Get all sorted timestamps, and only count the actions that actually ran
+    '''
+    timestamps = []
+    job_runs = job['runs']
+    for run in job_runs:
+        for action in run['runs']:
+            if action.get('start_time') and action.get('state') != 'scheduled':
+                timestamps.append(action.get('start_time'))
+    timestamps.sort()
```

Comment:
You shouldn't need to sort this, can't you just find the max?

Reply:
Oh right, I forgot python has this function.